doc/flatpak-metadata.xml

```diff
@@ -253,6 +268,54 @@
                                 Available since 0.3.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>host-os</option></term>
```

Comment:
I deliberately didn't call this `host-usr` because I thought that would be misleading. It's conceptually "/usr, plus everything that the /usr-merge would merge into it, plus the bits of `/etc` that are necessary if we want it to work properly".